Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5411,184 +5443,1373 @@ def panorama_show_location_ip_command():
+            'Action': ssl_decryption_rules.get('action'),
+            'Description': ssl_decryption_rules.get('description')
+        }
 
-        elif demisto.command() == 'panorama-unregister-ip-tag':
-            panorama_unregister_ip_tag_command()
+    headers = ['Name', 'UUID', 'Description', 'Target', 'Service', 'Category', 'Type', 'From', 'To', 'Source',
+               'Destination', 'Action', 'Source-user']
 
-        # Registered Users
-        elif demisto.command() == 'panorama-register-user-tag':
-            panorama_register_user_tag_command()
+    demisto.results({
+        'Type': entryTypes['note'],
+        'ContentsFormat': formats['json'],
+        'Contents': result,
+        'ReadableContentsFormat': formats['markdown'],
+        'HumanReadable': tableToMarkdown('SSL Decryption Rules', content, headers, removeNull=True),
+        'EntryContext': {"Panorama.SSLRule(val.UUID == obj.UUID)": content}
+    })
 
-        elif demisto.command() == 'panorama-unregister-user-tag':
-            panorama_unregister_user_tag_command()
 
-        # Security Rules Managing
-        elif demisto.command() == 'panorama-list-rules':
-            panorama_list_rules_command()
+def prettify_profile_rule(rule: Dict) -> Dict:
+    """
+    Args:
+        rule: The rule dictionary.
 
-        elif demisto.command() == 'panorama-move-rule':
-            panorama_move_rule_command()
+    Returns: Dictionary of the rule compatible with our standards.
 
-        # Security Rules Configuration
-        elif demisto.command() == 'panorama-create-rule':
-            panorama_create_rule_command()
+    """
+    pretty_rule = {
+        'Name': rule['@name'],
+        'Action': rule['action']
+    }
+    if 'application' in rule and 'member' in rule['application']:
+        pretty_rule['Application'] = rule['application']['member']
+    if 'file-type' in rule and 'member' in rule['file-type']:
+        pretty_rule['File-type'] = rule['file-type']['member']
+    if 'wildfire-action' in rule:
+        pretty_rule['WildFire-action'] = rule['wildfire-action']
+    if 'category' in rule and 'member' in rule['category']:
+        pretty_rule['Category'] = rule['category']['member']
+    elif 'category' in rule:
+        pretty_rule['Category'] = rule['category']
+    if 'severity' in rule and 'member' in rule['severity']:
+        pretty_rule['Severity'] = rule['severity']['member']
+    if 'threat-name' in rule and 'member' in rule['threat-name']:
+        pretty_rule['Threat-name'] = rule['threat-name']['member']
+    elif 'threat-name' in rule:
+        pretty_rule['Threat-name'] = rule['threat-name']
+    if 'packet-capture' in rule:
+        pretty_rule['Packet-capture'] = rule['packet-capture']
+    if '@maxver' in rule:
+        pretty_rule['Max_version'] = rule['@maxver']
+    if 'sinkhole' in rule:
+        pretty_rule['Sinkhole'] = {}
+    if 'sinkhole' in rule and 'ipv4-address' in rule['sinkhole']:
```

Comment:
no need to check `if 'sinkhole'` again, just check for ipv4 and ipv6 in the condition you added